### PR TITLE
test(artifacts test): AMI artifact test

### PR DESCRIPTION
**Pull request by enaydanov**
Trello: https://trello.com/c/AhDYBtIL/1660-migrate-scylla-ami-test-from-scylla-artifacts

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@bentsi, seem like the pylint duplicate code, was broken in some commit, and it shows in this one...
we might need to disable it, if it keep popping up in the wrong PRs

**Comment by fruch** [comment]
@enaydanov you should add the job in the relevant master folder

File: test-cases/artifacts/ami.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ami_id_db_scylla: 'ami-067787e9ea5433c78'
+aws_root_disk_size_db: 50
+cluster_backend: 'aws'
+instance_type_db: 'i3.large'
```
Comment (amoskong) [inline test-cases/artifacts/ami.yaml]: The instance_type should be configurable, and it supports to multiple instance types in parallel.

Comment (enaydanov) [reply test-cases/artifacts/ami.yaml]: Is it required to run same AMI on different instance types just to verify that AMI packaged well?

Comment (fruch) [reply test-cases/artifacts/ami.yaml]: this can be added to the pipeline only

Comment (fruch) [reply test-cases/artifacts/ami.yaml]: cause we have specific code per type in the AMI. different types are tested.

It's true we could cover it in some unit test easily. But we need the option to verify on different types

Comment (enaydanov) [reply test-cases/artifacts/ami.yaml]: done

**Comment by amoskong** [comment]
Hi @roydahan , do we want to backport this PR to 3.3 and replace existing ami-test (using scylla-artifact-tests)?

**Pull request merged**